index.html

```diff
@@ -0,0 +1,272 @@
+    <p class="services__decs">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</p>
+  </div>
+  <div class="services__item services__item--outlined">
+    <img src="images/icn3.JPG" class="services__img" />
+    <p class="services__title">High security to protect from piracy</p>
+    <p class="services__decs">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</p>
+  </div>
+ </div>
+
+ <section class="testimony">
+  <div class="testimony__item">
+    <p class="testimony__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</p>
+    <img src="images/one.png" class="testimony__img" />
+    <p class="testimony__name">Ron Wood</p>
+    <p class="testimony__role">CEO</p>
+  </div>
+  <div class="testimony__item">
+    <p class="testimony__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</p>
+    <img src="images/two.png" class="testimony__img" />
+    <p class="testimony__name">Ron Wood</p>
+    <p class="testimony__role">CEO</p>
+  </div>
+  <div class="testimony__item">
+    <p class="testimony__title">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</p>
+    <img src="images/three.png" class="testimony__img" />
+    <p class="testimony__name">Ron Wood</p>
+    <p class="testimony__role">CEO</p>
+  </div>
+
+</section>
+<footer class="footer">
+  <div class="content__background"></div>
+  <p class="title title--middle">
+    Are you ready to grow your <br/> business with us?
+  </p>
+  <p class="footer__subtitle">
+    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+  </p>
+  <button class="footer__button">
+    Free trial
+  </button>
+  <div class="footer-down">
+    <div class="footer-down__item footer-down__item--middle">
+      <div class="footer-down__list">
+          <p class="footer-down__title">
+              Company Info
+          </p>
+          <ul class="footer-down__links">
+              <li class="footer-down__link">About Us</li>
+              <li class="footer-down__link">Carrier</li>
+              <li class="footer-down__link">We are hiring</li>
+              <li class="footer-down__link">Blog</li>
+          </ul>
+      </div>
+      <div class="footer-down__list">
+          <p class="footer-down__title">
+              Company Info
+          </p>
+          <ul class="footer-down__links">
+              <li class="footer-down__link">About Us</li>
+              <li class="footer-down__link">Carrier</li>
+              <li class="footer-down__link">We are hiring</li>
+              <li class="footer-down__link">Blog</li>
+          </ul>
+      </div>
+      <div class="footer-down__list">
```

Comment:
Blocks inside blocks is not valid BEM. you can make `footer-down` an element of `footer` block instead.